definitions/ext-istio_service/definition.yml

```diff
@@ -22,6 +22,88 @@ synthesis:
     conditions:
     - attribute: metricName
       prefix: istio_tcp
+      
+  # Support for prometheus instrumentation provider
+  - identifier: "service_istio_io_canonical_name"
+    encodeIdentifierInGUID: true
+    name: "service_istio_io_canonical_name"
+    conditions:
+    - attribute: metricName
+      prefix: istio_request
+    tags:
+      service_istio_io_canonical_name:
+        entityTagNames: [label.service.istio.io/canonical-name, service_istio_io_canonical_name]
+      service_istio_io_canonical_revision:
+        entityTagNames: [label.service.istio.io/canonical-revision, service_istio_io_canonical_revision]
+      app:
+        entityTagNames: [label.app, app]
```

Comment:
I find this very weird, prefixing every tag with `label` is also true that it existed before for the existing istio entities but maybe can we avoid doing it for the new ones?

Wondering how needed this is really

Reply:
The other conditions won't apply when Prometheus remote write is used. The goal of this was to make the tagging consistent irrespective of the source method.